pytorch_lightning/loops/base.py

```diff
@@ -99,6 +99,53 @@ def connect(self, **kwargs: "Loop") -> None:
         Linked loops should form a tree.
         """
 
+    def replace(self, **loops: Union["Loop", Type["Loop"]]) -> None:
+        """Optionally replace one or multiple of this loop's sub-loops.
```

Comment:
should we document that it calls connect()?

Reply:
It's already mentioned below